travel_platform/izitravel_t.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
```

Comment:
If this folder contains __init__.py file, it's not important. I guess, root of this issue was highlighted in my next comment.